src/components/NavBar.tsx

```diff
@@ -0,0 +1,154 @@
+import { AppBar, Divider, Grid, MuiThemeProvider } from "@material-ui/core"
+import Box from "@material-ui/core/Box/Box"
+import Toolbar from "@material-ui/core/Toolbar/Toolbar"
+import Typography from "@material-ui/core/Typography/Typography"
+import KintoBlackLogo from "../images/kinto-black-logo.svg"
+import React, { useState } from "react"
+import styled from "styled-components"
+import { Link } from "gatsby"
+import { CallToActionButton } from "./Button"
+import KintoWhiteLogo from "../images/kinto-white-logo.svg"
+import { HorizontalSpacer } from "./Spacer"
+import {Drawer as MUIDrawer}  from "@material-ui/core"
+import ListItem from "@material-ui/core/ListItem/ListItem"
+import List from "@material-ui/core/List/List"
+import ListItemText from "@material-ui/core/ListItemText/ListItemText"
+import { textThemeDark } from "../../plugins/custom-mui-theme/theme"
+import ListItemIcon from "@material-ui/core/ListItemIcon/ListItemIcon"
+import LibraryBooksRoundedIcon from "@material-ui/icons/LibraryBooksRounded"
+import TuneRoundedIcon from "@material-ui/icons/TuneRounded"
+import LiveHelpRoundedIcon from "@material-ui/icons/LiveHelpRounded"
+import MailRoundedIcon from "@material-ui/icons/MailRounded"
+
+
+const Drawer = () => {
+  const itemList = [
+    { text: "Pricing", icon: <TuneRoundedIcon /> },
+    { text: "Documentation", icon: <LibraryBooksRoundedIcon /> },
+    { text: "Support", icon: <LiveHelpRoundedIcon /> },
+    { text: "Contact Us", icon: <MailRoundedIcon /> },
+  ]
+
+  return (
+    <MuiThemeProvider theme={textThemeDark}>
+      <MUIDrawer style={{width:"900px"}}>
+        <CallToActionButton buttonTitle="Login" variant="outlined" />
+        <CallToActionButton
+          buttonTitle="Signup Free"
+          variant="contained"
+          color="primary"
+        />
+        <List>
+          {itemList.map((item, index) => {
+            const { text, icon } = item
+
+            return (
+              <>
+                <ListItem button key={text}>
+                  <ListItemText primary={text} />
+                  <ListItemIcon>{icon}</ListItemIcon>
+                  <Divider />
+                </ListItem>
+              </>
+            )
+          })}
+        </List>
+      </MUIDrawer>
+    </MuiThemeProvider>
+  )
+        }    
+
+const StyledContainer = styled.div`
+  .active {
+    background-color: #fff;
+    a {
+      text-decoration: none;
+      color: #000000;
+    }
+  }
+  .inactive {
+    background-color: transparent;
+    box-shadow: none;
+  }
+`
+
+const StyledToolbar = styled(Toolbar)`
+  margin: 0;
+  padding: 0;
+  z-index: 999;
+
+  a {
+    text-decoration: none;
+    color: #fff;
+  }
+`
+
+const StyledBox = styled(Box)`
+  margin: 0px 16px;
```

Comment:
comment: you can have a root `styled.div` and contains all other classes for other components within this file, and use `className=""` in set the css there